test/Microsoft.TestPlatform.Common.UnitTests/ExtensionFramework/Utilities/RunSettingsProviderExtensionsTests.cs

```diff
@@ -134,13 +134,7 @@ public void UpdateRunSettingsNodeShouldAddNewKeyIfNotPresent()
         [TestMethod]
         public void UpdateTestRunParameterSettingsNodeShouldAddNewKeyIfNotPresent()
         {
-            var match = this.runSettingsProvider.GetTestRunParameterNodeMatch("TestRunParameters.Parameter(name=\"weburl\",value=\"http://localhost//abc\")");
-            var runSettingsWithTestRunParameters = "<?xml version=\"1.0\" encoding=\"utf-16\"?>\r\n<RunSettings>\r\n  <TestRunParameters>\r\n    <Parameter name=\"weburl\" value=\"http://localhost//abc\" />\r\n  </TestRunParameters>\r\n</RunSettings>";
-
-            this.runSettingsProvider.UpdateRunSettings("<RunSettings>\r\n  </RunSettings>");
-            this.runSettingsProvider.UpdateTestRunParameterSettingsNode(match);
-
-            Assert.AreEqual(runSettingsWithTestRunParameters, this.runSettingsProvider.ActiveRunSettings.SettingsXml);
+            this.CheckMatch("weburl", @"http://localhost//abc");
```

Comment:
That CheckMatch method does not do at all what I would expect by it's name. It seems as if it would check regex match but it does something totally different. Please consider renaming it.

Reply:
Done.